### ISSUE TITLE
TypeError: Cannot read property 'replace' of null error

### ISSUE THREAD

**Issue by cybersonic** [notwaldorf/tiny-care-terminal#165]
Hi there, when I run tiny-care-terminal I am getting the following error, I have set all the variables in my .zshrc file and have previously run it fine. Any help appreciated. 

```
TypeError: Cannot read property 'replace' of null
    at Box.Element.parseContent (/usr/local/lib/node_modules/tiny-care-terminal/node_modules/blessed/lib/widgets/element.js:364:8)
    at Box.Element.render (/usr/local/lib/node_modules/tiny-care-terminal/node_modules/blessed/lib/widgets/element.js:1839:8)
    at /usr/local/lib/node_modules/tiny-care-terminal/node_modules/blessed/lib/widgets/screen.js:738:8
    at Array.forEach (<anonymous>)
    at Screen.render (/usr/local/lib/node_modules/tiny-care-terminal/node_modules/blessed/lib/widgets/screen.js:735:17)
    at Box.<anonymous> (/usr/local/lib/node_modules/tiny-care-terminal/node_modules/blessed/lib/widgets/screen.js:1936:20)
    at Box.EventEmitter._emit (/usr/local/lib/node_modules/tiny-care-terminal/node_modules/blessed/lib/events.js:94:20)
    at Box.EventEmitter.emit (/usr/local/lib/node_modules/tiny-care-terminal/node_modules/blessed/lib/events.js:117:12)
    at Program.<anonymous> (/usr/local/lib/node_modules/tiny-care-terminal/node_modules/blessed/lib/widgets/screen.js:518:16)
    at Program.emit (events.js:198:13)
    at Program._bindMouse (/usr/local/lib/node_modules/tiny-care-terminal/node_modules/blessed/lib/program.js:703:10)
    at Program.<anonymous> (/usr/local/lib/node_modules/tiny-care-terminal/node_modules/blessed/lib/program.js:566:10)
    at Program.emit (events.js:198:13)
    at /usr/local/lib/node_modules/tiny-care-terminal/node_modules/blessed/lib/program.js:418:15
    at Array.forEach (<anonymous>)
    at ReadStream.input.on.input._dataHandler (/usr/local/lib/node_modules/tiny-care-terminal/node_modules/blessed/lib/program.js:416:23)
```

- **Node Version**: v10.16.0
- **tiny-core-terminal version**: `tiny-care-terminal@1.7.0 | MIT | deps: 19 | versions: 20`
- **Shell**: `zsh`
- **Terminal Program**: `terminal`
- **Operating System**: `Mac`

**Comment by mojoaxel** [comment]
@cybersonic Sorry to hear that! I'll have a look today, propably error-handling should be inproved!
What environment variables have you set?

**Comment by nomoon** [comment]
I'm pretty sure this is caused by the new random art feature. I didn't have any `TTC_SAY_BOX` variable being set in my environment and also got this error. Once I put `export TTC_SAY_BOX='RANDOM'` into my shell, everything worked.

**Comment by mojoaxel** [comment]
This is fixes in v1.7.1 :rocket: 
Sorry y'all for the problems! :crying_cat_face:
